static/css/jira-configuration.css

```diff
@@ -0,0 +1,43 @@
+.jiraConfigurationTable {
```

Comment:
Have these css changes been tested cross-browser? In just one browser? What's our check for compatibility?

Reply:
I've checked in several browsers. Original screen shot is from Chrome. 

Safari:

<img width="1540" alt="safari" src="https://user-images.githubusercontent.com/37155488/116947756-e7590000-acc0-11eb-80f9-64b53ad0ab17.png">

Firefox:

<img width="1551" alt="firefox" src="https://user-images.githubusercontent.com/37155488/116947763-eaec8700-acc0-11eb-99da-c5604c8ccd86.png">